CMakeLists.txt

```diff
@@ -1816,6 +1816,15 @@ if(CONFIG_CHECK_INIT_PRIORITIES)
     )
 endif()
 
+add_custom_target(
+  initlevels
```

Comment:
we should not introduce a build target which will fail in some setups when executed by the user.

In such cases, creation of the build target should be properly safe guarded.

Reply:
Got it, guarded, kept it in a separate commit, easy to rollback if that gets fixed.